bundle/manifests/orchestrator-operator.clusterserviceversion.yaml

```diff
@@ -15,9 +15,15 @@ metadata:
               "enabled": false,
               "namespace": ""
             },
+            "networkPolicy": {
+              "rhdhNamespace": "rhdh-operator"
+            },
```

Comment:
should this be part of this PR?

Reply:
> should this be part of this PR?

Cleaned ip :)

Reply:
Ah I see what happened. The problem is that we no longer generate the bundle as part of the build because Konflux forces us to run hermetic builds and we can't retrieve opm at building time. So we're left with sed/awk to make changes on image and version values. And when we have additional changes like new fields, those are not populated. But if we run the `make bundle`, then these fields now show up in the bundle like something new we changed.

Let me raise a PR now and fix this before yours is merged, then your PR won't have these artifacts.

Reply:
Opened #428 that runs `make bundle` in the existing code. Once it is merged, @jianrongzhang89 please rebase and update the PR, it should have less changes due to running `make bundle` on your side